stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Channels.kt

```diff
@@ -282,6 +282,42 @@ class Channels(val client: ChatClient, val channelClient: ChannelClient) {
      * @see <a href="https://getstream.io/chat/docs/channel_update/?language=kotlin">Updating a Channel</a>
      */
     inner class UpdatingAChannel {
+
+        /**
+         * @see <a href="https://getstream.io/chat/docs/android/channel_update/?language=kotlin#partial-update">Partial Update</a>
+         */
+        fun partialUpdate() {
+            // Here's a channel with some custom field data that might be useful
+            val channelClient = client.channel(channelType = "messaging", channelId = "general")
+
+            channelClient.create(
+                members = listOf("thierry", "tomasso"),
+                mapOf(
+                    "source" to "user",
+                    "source_detail" to mapOf("user_id" to 123),
+                    "channel_detail" to mapOf(
+                        "topic" to "Plants and Animals",
+                        "rating" to "pg"
+                    )
+                )
+            ).execute()
```

Comment:
We normally use `enqueue` in all our docs to promote the usage of async calls by default + show an example of error handling